bears/c_languages/CPPLintBear.py

```diff
@@ -9,7 +9,9 @@
         output_regex=r'.+:(?P<line>\d+): (?P<message>.+)')
 class CPPLintBear:
     """
-    Checks the code with ``cpplint`` on each file separately.
+    Check C++ code for Google's C++ style guide.
```

Comment:
no actually I disagree, it checks it for compliance to that guide, it's not using it